.github/workflows/on-release.yaml

```diff
@@ -0,0 +1,43 @@
+name: Deploy to Quay.io
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    env:
+      NODE_OPTIONS: --max-old-space-size=8192
+      QUAY_IO_REGISTRY_OWNER: edge-infrastructure
+    steps:
+      - name: Define additional environment variables
+        uses: actions/github-script@v6
+        with:
+          script: |
+            core.exportVariable('REACT_APP_VERSION', '${{ github.ref_name }}');
+            core.exportVariable('REACT_APP_GIT_SHA', '${{ github.sha }}'.slice(0,8));
+
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Build the assisted-ui image
+        id: build-image
+        uses: redhat-actions/buildah-build@v2
+        with:
+          image: assisted-installer-ui
```

Comment:
is it possible changing the repo name to ``assisted-installer-ui`` as well?
not a blocker for this PR of course, just for consistency

Reply:
I had that in mind, but since I wasn't sure of the impact this change might introduce I didn't want to do that at this moment.
I will put a task for this apart.